qa/L0_e2e/generate_example_model.py

```diff
@@ -133,6 +183,26 @@ def train_lightgbm_regressor(data, targets, depth=25, trees=100):
     return model
 
 
+def train_sklearn_regressor(data, targets, depth=25, trees=100):
```

Comment:
Not a big deal and probably worth of a cleanup follow up PR, but these functions could be condensed to a single ones with the classifier ones if the class is passed as parameter

Reply:
Yeah, I think that makes good sense! @hcho3 expressed some interest in more general refactoring of these test scripts post-21.06, so if it's all right by you, I think I'll leave this for that effort.

Reply:
Also added it as an issue so we don't forget: https://github.com/triton-inference-server/fil_backend/issues/72